src/dataset-builder/DatasetBuilderUtils.ts

```diff
@@ -237,6 +237,9 @@ export const HighlightConceptName = ({ conceptName, searchFilter }): ReactElemen
   ]);
 };
 
-export const displayParticipantCount = (count: number): string => {
+export const displayParticipantCount = (count: number | undefined): string => {
```

Comment:
To me, this name doesn't describe what the method is doing because it doesn't display the participant count. It either returns the count as a string or the "less than 20" string. Maybe `formatCount()` would be better?